### ISSUE TITLE
Custom php template not needed for Find Us 

### ISSUE THREAD

**Issue by rcass** [mariesarabia/Project-4-Inhabitent#3]
No need to have a custom php template for the Find Us page since it is not generating any special content or has a different layout.

The find us page is an example of a basic page (there for the php file that would be used is page.php) but really no php modification is necessary.

You will just need to style the page and be sure to add the Find Us content via the WYSIWYG editor.

Let me know if you have any questions! 👍 

TL;DR

- [x] Delete find-us.php

**Comment by mariesarabia** [comment]
Find-us.php deleted
thanks
